lib/framework/framework.dart

```diff
@@ -131,6 +131,14 @@ class Framework {
   void clearError() {
     querySelector('#error-container').children.clear();
   }
+
+  void toast(String message, {String title}) {
+    final Toast t = new Toast(title: title, message: message);
```

Comment:
nit: Take advantage of optional new and omit types on the LHS when they don't add value.

Reply:
avoid abbreviations when reasonable. Call this `toast` instead of `t`